src/System.Management.Automation/CoreCLR/CorePsExtensions.cs

```diff
@@ -1069,7 +1069,18 @@ private static string InternalGetFolderPath(SpecialFolder folder)
                     break;
                 case SpecialFolder.LocalApplicationData:
                     folderPath = System.IO.Path.Combine(envHome, ".config");
-                    if (!System.IO.Directory.Exists(folderPath)) { System.IO.Directory.CreateDirectory(folderPath); }
+                    if (!System.IO.Directory.Exists(folderPath)) 
+                    {
+                        try
+                        {
+                            System.IO.Directory.CreateDirectory(folderPath); 
+                        }
+                        catch (UnauthorizedAccessException)
+                        {
+                            // some scenarios run as accounts without filesystem access
```

Comment:
Oh, sorry, I again don't understand the comment for the first time. Could you make it more clear?